draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -294,12 +294,12 @@
         	</artwork>
 		</figure>
 	
-		<t><xref target="fig:fec-in"></xref> presents an architecture where FEC is within the transport. The repair symbols are sent within what the congestion window allows, such as in <xref target="CTCP"/>.</t>
+		<t><xref target="fig:fec-in"></xref> presents an architecture where FEC operates within the transport. The repair symbols are sent within what the congestion window allows, such as in <xref target="CTCP"/>.</t>
 	</section>
 
 	<section anchor="sec:fec-in-discu" title="Discussion">
 		<t>The advantage of this approach allows a joint optimization between the CC and the FEC. Moreover, the transmission of repair symbols does not add congestion in potentially congested networks but helps repair lost packets (such as tail losses).</t>
-		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit or preferably sending repair symbols instead of the following packets in the send buffer could help to reduce the transfer time. This is because it could avoid a silent period between the transmission of the last packet and firing the retransmission of lost packets.</t>
+		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit could help to reduce the transfer time. This is because it could avoid a silent period between the transmission of the last packet and firing the retransmission of lost packets.</t>
```

Comment:
I didn't understand the idea of "or preferably sending repair symbols instead of the following packets in the send buffer", but I also think it only makes this text more complicated. The next sentence talks about a silent period, so it's better to keep it focused on this only.

Reply:
This discussion is outdated now.